### PR TITLE
update URI endpoints

### PR DESCRIPTION
**Pull request by disrupted**
fixes Vimeo (https://github.com/klattimer/KodiPlay/issues/4) and SoundCloud, updates YouTube endpoint

### CODE REVIEW

**Comment by disrupted** [comment]
any comments @klattimer ?

**Comment by disrupted** [comment]
**ping** @klattimer, do you copy?

File: Info.plist [hunk 1]
```diff
@@ -42,7 +42,7 @@
 	<key>Description</key>
 	<string>Play videos to Kodi media centre </string>
 	<key>DeveloperIdentifier</key>
-	<string>SYD7AF6B55</string>
+	<string>4853CED358</string>
 	<key>ExtensionInfoDictionaryVersion</key>
```
Comment (frankosterfeld) [inline Info.plist:46]: These Info.plist changes should probably not be part of the patch?

**Comment by disrupted** [comment]
yea sorry about that, should have reverted that line change earlier.. → please merge https://github.com/klattimer/KodiPlay/pull/6
